source/includes/driver-compatibility-caveat.rst

```diff
@@ -0,0 +1,6 @@
+.. note::
+
+   The compatibility matrices show which versions of the driver are
+   recommended for use with a given major release. Older driver versions
+   may be able to connect to a version of MongoDB which is not officially
+   supported, but not all features will be available.
```

Comment:
This second sentence needs work.  Your sentence in effect states:

Older driver version can connects to unsupported versions of MongoDB but not all features will be available.

Is that what you meant?